src/components/MonthlyGood/AutomaticallySupportNotice.vue

```diff
@@ -32,6 +32,8 @@
 </template>
 
 <script>
+import KvContentfulImg from '@/components/Kv/KvContentfulImg';
```

Comment:
Please convert this image usage to use the new version of `KvContentfulImg` out of kv-components.